cpp/include/DataStorm/DataStorm.h

```diff
@@ -1,18 +1,16 @@
 //
 // Copyright (c) ZeroC, Inc. All rights reserved.
 //
-#pragma once
 
-#include <DataStorm/Config.h>
+#ifndef DATASTORM_DATASTORM_H
+#define DATASTORM_DATASTORM_H
 
-#include <Ice/Ice.h>
```

Comment:
I was under the impression that DataStorm used Ice only as an implementation detail and did not expose the Ice API... this is not the case. And this makes merging DataStorm into Ice easier: we can use Ice APIs / types in the DataStorm API.